saleor/checkout/tests/test_calculations.py

```diff
@@ -67,6 +71,23 @@ def tax_data(checkout_with_items, checkout_lines):
     )
 
 
+@pytest.fixture
+def checkout_with_item_and_gift_card(checkout_with_item, gift_card):
+    # Add the gift card to the checkout
+    checkout = checkout_with_item
+    checkout.gift_cards.add(gift_card)
+    checkout.save()
+    return checkout
+
+
+def gift_card(db):
```

Comment:
I don't think that this is used anywhere. It looks like a fixture but without proper decorator. 
Also `GifCard` does not have field like `balance`. IMHO we can safely drop it